Applications/Hermes/hermesflaskserver.py

```diff
@@ -0,0 +1,177 @@
+import os
+import hermesobject as hermesobject
+import ConfigParser
+import json
+import base64
+import cPickle
+
+from flask import Flask
+from flask import request
+
+# Open configuration INI
```

Comment:
**Non-Blocking**
My Telemetry application shows [how to import safely](https://github.com/kb1lqc/Faraday-Software/blob/c26a906c62d08d484ef6804687d08dee798ae940/Applications/Telemetry/telemetry.py#L42)